Makefile.dep

```diff
@@ -634,6 +634,10 @@ ifneq (,$(filter puf_sram,$(USEMODULE)))
   FEATURES_REQUIRED += puf_sram
 endif
 
+ifneq (,$(filter can_stm32,$(USEMODULE)))
+  USEMODULE += xtimer
```

Comment:
why? I didn't find any usage of timer, or did I miss something?

Reply:
probably left-over, I'll remove